src/wrapper.py

```diff
@@ -86,25 +88,29 @@ def bootstrap(_return=0):
                         requests.get(url=master_url + '/stop')
                         logger.info('Load test is stopped.')
 
-                        logger.info('Downloading reports...')
                         time.sleep(4)
+
+                        logging.info('Creating report folder.')
                         report_path = os.path.join(os.getcwd(), 'reports')
-                        os.makedirs(report_path)
+                        if not os.path.exists(report_path):
+                            os.makedirs(report_path)
 
+                        logger.info('Creating reports...')
                         for _url in ['requests', 'distribution']:
                             res = requests.get(url=master_url + '/stats/' + _url + '/csv')
                             with open(os.path.join(report_path, _url + '.csv'), "wb") as file:
                                 file.write(res.content)
 
-                            if _url == 'distribution': continue
+                            if _url == 'distribution':
+                                continue
                             res = requests.get(url=master_url + '/stats/' + _url)
                             with open(os.path.join(report_path, _url + '.json'), "wb") as file:
                                 file.write(res.content)
 
                         res = requests.get(url=master_url + '/htmlreport')
                         with open(os.path.join(report_path, 'reports.html'), "wb") as file:
                             file.write(res.content)
-                        logger.info('Reports have been successfully downloaded.')
```

Comment:
well, they are actually downloaded from master

Reply:
hm.. actually it is not downloaded, it read the content and save it in file. Anyway I use logger just to debug the step.

Reply:
I just mean they're "obtained from the network" rather than "created"